apiary.apib

```diff
@@ -190,6 +190,64 @@ This route is the destination URL for password reset confirmation
             expiry: 1489009792
             uid: test@test.com
 
+## Topics Collection
+
+### Get Topics
+
++ Request (application/json)
+    + Headers
+            access-token: 0JfjvKLv7NR3_d10350u2w
+            client: pBx8B5nowYy3L-EXueP6nQ
+            uid: test@test.com
+            token-type: Bearer
+
++ Response 200 (application/json)
+    + Body
+
+            [
+              {
+                  "id": 1,
+                  "label": "Football"
+              },
+              {
+                  "id": 2,
+                  "label": "Travel"
+              },
+              {
+                  "id": 3,
+                  "label": "Politics"
+              },
+              {
+                  "id": 4,
+                  "label": "Art"
+              },
+              {
+                  "id": 5,
+                  "label": "Dating"
+              },
+              {
+                  "id": 6,
+                  "label": "Music"
+              },
+              {
+                  "id": 7,
+                  "label": "Movies"
+              },
+              {
+                  "id": 8,
+                  "label": "Series"
+              },
+              {
+                  "id": 9,
+                  "label": "Food"
+              }
```

Comment:
Don't need to put all these topics in here. Just put one, the ``` [ ] ``` implies it's a list.